data-resource/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+mod id;
```

Comment:
Probably we can just use `lib.rs` until we'll really have something else except ids.

Reply:
Sure!
I also think we can port https://github.com/ARK-Builders/arklib/pull/90 to `ark-rust` when we have this PR merged